Website/src/rendering/src/components/PageContent/Promo.tsx

```diff
@@ -26,11 +28,16 @@ const Promo = ({ fields }: PromoProps): JSX.Element => {
   return (
     <section className={sectionCssClasses}>
       <div className={positionCssClasses}>
-        <div className="section__promo__content__card">
+        <div
+          className="section__promo__content__card"
+          style={{ backgroundImage: 'url(' + fields.backgroundImage?.value?.src + ')' }}
+        >
           <Text tag="h5" field={fields.subtitle} />
           <Text tag="h2" field={fields.title} />
           <RichText field={fields.content} />
-          <Link field={fields.callToActionLink} className="btn--main btn--main--round" />
+          {fields.callToActionLink.value.text && (
```

Comment:
Please avoid mixing logic and markup inside the main JSX return. Please implement this condition with a const like the one in `FullImageSection.tsx`. Thanks